### ISSUE TITLE
Model RNG seed not working with keras 2.3.0

### ISSUE THREAD

**Issue by budach** [budach/pysster#8]
When training two pysster Model objects with the same seed, model weights are not equal anymore with keras 2.3.0. Respective tests have been disabled for now. Might not need a fix, since we want to switch to tf.keras anyway.

**Comment by budach** [comment]
Well, there seem to be more issues, like a decrease in predictive performance for my test models. I am blacklisting this keras version.

**Comment by budach** [comment]
No changes with keras 2.3.1.
